spec/api/viewing_all_applications_spec.rb

```diff
@@ -0,0 +1,34 @@
+# TODO: import this automagically for /api folder
+require 'rails_helper'
+
+describe 'GET applications', type: :request do
+  before do
+    headers = { "ACCEPT" => "application/json" }
+    get '/api/applications', headers: headers
+  end
+
+  it 'responds with a success code' do
+    expect(response).to have_http_status(:ok)
+  end
+
+  it 'contains three applications' do
+    expect(JSON.parse(response.body).count).to eq(3)
```

Comment:
Can we DRY `JSON.parse(response.body)`? 🚀

Reply:
Good shout - addressed in https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/33/commits/fd576b45957966998d50404349ef6a0f10aaabdc